misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -97,24 +82,31 @@ class PickStepButton extends PureComponent {
   }
 }
 
+const mapStateToProps = (state, ownProps) => {
+  const { wfProcessId, activityId, lineId, stepId } = ownProps;
+
+  return {
+    stepState: state.wfProcesses_status[wfProcessId].activities[activityId].dataStored.lines[lineId].steps[stepId],
+    appId: state.applications.activeApplication ? state.applications.activeApplication.id : null,
```

Comment:
is the `appId` still needed? i mean, we know that we are talking about picking.....

Reply:
Nope, we have `appId` in the url now since every app has it's own path